solutions/swb-reference/RSW.openapi.json

```diff
@@ -2299,6 +2299,22 @@
         "description": "List Users",
         "operationId": "listUsers",
         "parameters": [
+          {
+            "name": "paginationToken",
+            "in": "query",
+            "schema": {
+              "type": "string",
+              "example": ""
```

Comment:
do we need an example here?

Reply:
no this is how the other APIs have it in this file